main/src/cgeo/geocaching/filter/CountryFilterFactory.java

```diff
@@ -10,38 +10,28 @@
 import androidx.annotation.NonNull;
 
 import java.util.ArrayList;
-import java.util.HashMap;
+import java.util.LinkedHashMap;
 import java.util.List;
 import java.util.Map;
+import java.util.Objects;
 
 public class CountryFilterFactory implements IFilterFactory {
```

Comment:
@JakeDot note that the filter system has changed completely since this class was created. As of now the framework it is based on (eg IFilterFactory) des not exist any more.